src/source/source_simple_led.ts

```diff
@@ -12,7 +12,8 @@ import {
 export const source_simple_led = source_simple_diode.extend({
   ftype: z.literal("simple_led"),
   color: z.string().optional(),
-  wavelength: z.string().optional(),
+  wave_length: z.string().optional(),
+  sch_val_label: z.string().optional(),
```

Comment:
I don't think this belongs on the source component since it's related to the schematic display